package.json

```diff
@@ -13,6 +13,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "arranger-server-jaserud": "1.0.1",
```

Comment:
I published arranger to my own npm just for testing and to make sure docker is building, will point to actual arranger once merge